src/documentation/Supported_Languages.ipynb

```diff
@@ -2677,6 +2677,1147 @@
     "Note that SoS uses the feather modules in Python and R to exchange Python Pandas `Dataframe` and R `data.frame` so this module is required if you would like to exchange data frames between these two languages."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Ruby "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Basic data types recognised in Ruby are similar with Python's data types and there is a one-to-one correspondence for these types.\n",
+    "\n",
+    "The convertion of datatype from SoS to Ruby (e.g. `%get` from Ruby) is as followings:\n",
+    "\n",
+    "  \n",
+    "  | Python  |  condition |   Ruby |\n",
+    "  | --- | --- |---|\n",
+    "  | `None` | |    `nil` |\n",
+    "  | `boolean` |   | `TrueClass or FalseClass` |\n",
+    "  | `integer` |  |  `Integer` |\n",
+    "  | `float` |  |  `Float` |\n",
+    "  | `complex` |  |  `Complex` |\n",
+    "  | `str` |  | `String` |\n",
+    "  | Sequence (`list`, `tuple`, ...) |  |  `Array` |\n",
+    "  | `set` |  |  `Set` |\n",
+    "  | `dict` |  |  `Hash` |\n",
+    "  | `range` |  |  `Range` |\n",
+    "  | `numpy.ndarray` |  | `Array` |\n",
+    "  | `numpy.matrix` |  | `NMatrix` |\n",
+    "  | `pandas.Series` |  | `Hash` |\n",
+    "  | `pandas.DataFrame` |  | `Daru::DataFrame` |\n",
+    "\n",
+    "  Python objects in other datatypes are transferred as string `\"Unsupported datatype\"`."
```

Comment:
I would add, perhaps also in other places, something like:

as string "Unsupported datatype". Please [let us know](git hub issue tracker) if there is a natural corresponding data type in Ruby to convert this data type.

Reply:
There are too many such cases. Perhaps you can add a general comment before specific languages.